src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/AccountDelegate.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans.xsd">
+<!--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2023 Kuali, Inc.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+  <bean parent="DataDictionaryBeanOverride">
+       <property name="beanName" value="AccountDelegate-lookupDefinition" />
+       <property name="fieldOverrides">
+           <list>
+               <bean parent="FieldOverrideForValueReplace">
+                   <property name="propertyName" value="multipleValuesResultSetLimit" />
+                   <property name="value" value="10000" />
```

Comment:
Since you're just overriding the value of a single easy-to-reach property, and since the target bean follows KualiCo's parent bean convention, It might be better to just override the `AccountDelegate-lookupDefinition` bean directly and put the property override there.

Also, I'm wondering whether 10000 is small enough. I know that the regular Account Delegate maintenance document allows up to 10000 changes, but that limit represents the doc's number of delegates multiplied by the doc's number of accounts (meaning the document itself likely won't display all 10000 of the resulting changes). Would 1000 be a more reasonable limit, like what our custom `AccountGlobalSearch` lookup uses? Or do the functionals absolutely need the 10000 limit instead?

Reply:
Cathy has requested that the value be 10000 specifically.
Looking into overriding the lookupDefinition directly.